packages/origin-backend-core/src/User.ts

```diff
@@ -1,5 +1,13 @@
 import { IOrganization } from './Organization';
 
+type Currency = string;
+
+export interface IAutoPublishConfig {
+    enabled: boolean;
+    currency: Currency;
+    priceInCents: number;
```

Comment:
Is this still needed? Will certificates still be autopublished in the new exchange?

Reply:
Yes, we will retain such a functionality.